src/createBoundingBox.js

```diff
@@ -0,0 +1,82 @@
+//Update popper position
+module.exports.updatePopperObjectPosition = function (target) {
+    var popper = target.scratch('popper');
+    popper.scheduleUpdate();
+    return popper;
+};
+
+//Return dimensions
+module.exports.getPopperObjectDimensions = function (target, userOptions) {
+    //Set Defaults
+    var width = target.width();
+    var height = target.height();
+
+    //Overide with the outer-dimensions if a bounding box is provided
+    if (userOptions.boundingBox) {
+        width = userOptions.boundingBox.w;
+        height = userOptions.boundingBox.h;
+    }
+
+    //Override defaults with 3 if popper is called on core
+    if(userOptions.core){
```

Comment:
You should instead be setting a boundingBox for the core case.  No function but the top-level `cy.popperRef()` or `ele.popperRef()` should have node/edge/core-specific code.

Reply:
Removed